.github/workflows/build_release.yml

```diff
@@ -11,9 +11,25 @@ jobs:
     name: build
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
       with:
         submodules: true
+    
+    - name: Make setup.sh executable
+      run: chmod +x debian/setup.sh
+
+    - name: Build Debian Package
+      working-directory: ./debian
+      run: ./setup.sh ${{ github.event.release.tag_name }}
+
+    - name: Upload debian package to release
+      uses: softprops/action-gh-release@v2
```

Comment:
I would like to not have more dependencies here. The deb package can be uploaded with:
```
      - uses: actions/upload-artifact@v4
        with:
          name: capstone.deb
          path:<path-t0-source>/capstone.deb
```
Or does `action-gh-release` also take care of publishing it in the debian repos?

Reply:
This only uploads the debian package to release, this doesn't do anything with the debian apt repos.